### PR TITLE
Add polyline decoding by default, add new test fixture that tests tha…

### LINKED ISSUE #26: Request and decode polyline encoding
`geometry=polyline` results in _much_ smaller response sizes. We’d need to decode the polyline strings into the same array of coordinates we currently expose to client code.

/cc @bsudekum

### PR DESCRIPTION
**Pull request by tmcw**
…t ability.

Fixes #26

* Adds `Polyline` pod dependency

cc @1ec5 for the review

### CODE REVIEW

File: Podfile [hunk 1]
```diff
@@ -4,5 +4,6 @@ use_frameworks!
 pod 'NBNRequestKit', :git => 'https://github.com/1ec5/RequestKit.git', :branch => 'mapbox-podspec'
 
 target 'MapboxDirectionsTests' do
+  pod 'Polyline', '~> 3.0'
```
Comment (1ec5) [inline Podfile]: This dependency needs to be added to the implicit target above, like NBNRequestKit. This section is for dependencies specific to the test bundle.

**Comment by tmcw** [comment]
@1ec5 fixed the pod mistake - can you add me to BitRise so I can debug the test failure?

**Comment by 1ec5** [comment]
Polyline.framework needs to be added as a dependency of the MapboxDirections target in the Xcode project editor. Additionally, the Polyline pod needs to be added as a dependency in the podspec.

**Comment by tmcw** [comment]
@1ec5 I think I have it set that way?

* https://github.com/mapbox/MapboxDirections.swift/pull/27/files#diff-bdad080b027f4baa083979ec946fc55cR12 - Polyline.framework in the project
* https://github.com/mapbox/MapboxDirections.swift/pull/27/files#diff-4a25b996826623c4a3a4910f47f10c30R5 polyline as a pod

And visually:

2016-04-06 at 2 58 pm

**Comment by tmcw** [comment]
:tada: tests pass! Anything else @1ec5 ?

**Comment by 1ec5** [comment]
Nope, merge away!

**Comment by friedbunny** [comment]
🎉

**Pull request merged**